packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-custom-pwd-recovery-mfa/main/java/emailtemplate.md

```diff
@@ -0,0 +1,11 @@
+If you're using the Sign-In Widget sample app or the SDK sample app use `http://localhost:8080/magic-link/callback?otp=${oneTimePassword}&state=${request.relayState}`. The following snippet shows how the URL looks in the HTML message:
```

Comment:
Nice - changed to ${yourOktaDomain}

Reply:
Are you sure using ${yourOktaDomain} is accurate here? This is the Okta org for the company. `localhost:8080` is the app's URL.

Reply:
Good point - will change to ${yourAppDomain}. Oddly, this will be the only use of it in the whole of the doc site but there is no alternative I can see.